__tests__/app.test.js

```diff
@@ -77,3 +77,32 @@ describe('GET /api', () => {
       });
   });
 });
+
+describe('GET /api/articles', () => {
+  test('200: returns an array of articles objects', () => {
+    return request(app)
```

Comment:
we just need to add another test to check they come back ordered by date.  Check out jest sorted!